requirements/requirements.common.txt

```diff
@@ -4,88 +4,89 @@
 #
 #    pip-compile --output-file=requirements/requirements.common.txt setup.py
 #
+git+https://github.com/pulp/pulp_ansible.git@master#egg=pulp_ansible  # via galaxy-ng (setup.py)
```

Comment:
This is wrong. `requirements.txt` is used to lock versions installed in docker containers. Installing from SCM must include a fixed reference (e.g. tag or commit SHA).

Reply:
pip-compile really seems to want there to be a tag, but at the moment, there is no pulpcore 3.6.* tag. 
Spec'ing the latest hash from master gives warnings:

WARNING: Requested pulpcore>=3.0 from git+https://github.com/pulp/pulpcore.git@64b9fb994704691173258cd3f0fecfca70a2bff5#egg=pulpcore (from galaxy-ng (setup.py)), but installing version 3.6.0.dev0

Reply:
That's likely because `pulpcore>=3.0` constraint does not include pre-release versions.